### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by tmf97**
### Summary

This PR cleans up logging a bit

### Testing
You should be able to run `main.py` on your computer with the venv setup and see that logging works

### Notes
I kinda realized how pointless the `log_error` function in log.py is but haven't gotten rid of all uses of it. I'll get to it eventually

### Comments
- [x] Add an x between the brackets if you commented your code well!

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -31,13 +31,12 @@
...
-logger = get_log()
-
-
 class MainSatelliteThread(Thread):
+    flight_mode: FlightMode
+
     def __init__(self):
         super().__init__()
-        logger.info("Initializing...")
+        logging.info("Initializing...")
```
Comment (sjz38) [inline main.py:39]: I see you changed the `logger` calls to `logging`. Should this be done everywhere else in code (eventually)?

**Review by sjz38 — APPROVED** [review]
Everything looks good

File: main.py (hunk 1 above)
Comment (tmf97) [reply main.py:39]: Done

**Comment by sjz38** [comment]
👍

**Comment by tmf97** [comment]
yeet

**Pull request merged**